storage/rocksdb/ha_rocksdb.cc

```diff
@@ -5252,6 +5418,37 @@ static void rocksdb_update_optimizer_costs(OPTIMIZER_COSTS *costs)
   costs->row_copy_cost=      0.00006087;
```

Comment:
One minor thing, it is likely these will need recalculating. That doesn't have to be in this PR though.